.travis.yml

```diff
@@ -2,6 +2,7 @@ language: node_js
 before_install:
 - npm install -g npm@latest
 node_js:
-- '8.0'
 - '10.0'
 - '12.0'
+- '12.11'
+- '14'
```

Comment:
According to our offline discussion yesterday, we can probably set these to `lts/*` and `stable` and be done with it 🤷